tests/docker_common.py

```diff
@@ -59,21 +60,26 @@ def diff_dict(left, right):
 
 def event_test(agent, name, pre_func=None, post_func=None, diff=True):
     req = json_data(name)
-    resp_valid = json_data(name + '_resp')
+    valid_resp_file = json_data(name + '_resp')
+    valid_resp = JsonObject.unwrap(valid_resp_file)
 
     if pre_func is not None:
         pre_func(req)
 
     resp = agent.execute(req)
     if post_func is not None:
-        post_func(req, resp)
+        insp = inspect.getargspec(post_func)
+        if len(insp.args) == 3:
```

Comment:
@alena1108  This is actually inspecting the signature of the post function that we pass in when we call this function from a test, ie:
```
event_test(agent, schema, pre_func=pre, post_func=post)
```
Up til this point, the signature of this function has had to be:
```
def post(req, resp):
```
but I added an optional third argument so that the signature _can_ be:
```
def post(req, resp, valid_resp):
```
...the way I made this optional is doing this inspect.getargspect() and looking at the number of args in the function.